javascripts/content_scripts/privly.js

```diff
@@ -299,8 +298,8 @@ var privly = {
       var notUpdated = [];
       elements.forEach(
         function(a){
-          if ( ! a.textContent.indexOf("privlyInject1") > 0 // Optimization
-            || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
+          // Optimization
+          if (  a.textContent.indexOf("privlyInject1") < 0 || ! privly.correctIndirection.testAndCopyOver(a, a.textContent) ) {
```

Comment:
This should be `<=` to maintain the same semantics. Also, it is preferable to have this on two lines so it doesn't break the width limitations.